pkg/migration-tool/planner/planner.go

```diff
@@ -161,19 +162,19 @@ func (p *Plan) createBlock(mint, maxt int64) (reference *Block, err error) {
 		return nil, fmt.Errorf("create-block: %w", err)
 	}
 	id := p.blockCounts.Add(1)
-	timeRangeInMinutes := (maxt - mint) / time.Minute.Milliseconds()
+	timeRangeInMinutes := (maxt - mint) / minute
 	percent := float64(maxt-p.config.Mint) * 100 / float64(p.config.Maxt-p.config.Mint)
 	if percent > 100 {
 		percent = 100
 	}
-	baseDescription := fmt.Sprintf("progress: %.3f%% | block-%d time-range: %d mins | mint: %d | maxt: %d", percent, id, timeRangeInMinutes, mint, maxt)
+	baseDescription := fmt.Sprintf("progress: %.3f%% | block-%d time-range: %d mins | mint: %d | maxt: %d", percent, id, timeRangeInMinutes, mint/second, maxt/second)
 	reference = &Block{
 		id:                    id,
 		pbarDescriptionPrefix: baseDescription,
 		pbar: progressbar.NewOptions(
```

Comment:
How do these 2 progressbars interact? The one here and the one in client.go? Do they simply override one another? How will this work when we have multiple blocks in-flight at the same time?

Reply:
They override each other. But I do fear that if the reader is puling multiple blocks simulatenously, then it can be an issue as I don't think that `io.Copy` can do copying in a concurrent state. So maybe, if the user chooses to have concurrent fetches then we disable the transfer-rate and only show transfer-rate if fetching concurrent blocks is disabled. 
(as if concurrent fetches and concurrent flushes are enabled, then the progress bar itself becomes more interactive).
WDYT?

Reply:
@cevian does it look good?

Reply:
for now, yes